src/ImageSequencer.js

```diff
@@ -1,5 +1,6 @@
 if (typeof window !== 'undefined') {isBrowser = true}
 else {var isBrowser = false}
+require('./util/prevStepAPI.js')
```

Comment:
Should this maybe be called `getStep.js`?

Reply:
Oh, we can call this whatever you say, I just put it whatever came to my mind, sorry I'm not very good at naming things😅